### PR TITLE
fix symfony integration doc

### PR DESCRIPTION
**Pull request by dbu**
fix some typos

### CODE REVIEW

File: phpcr-shell/installation.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ Simply add the shell to your ``composer.json`` file
         ...
         require: {
             ...
-            "phpcr-shell": "<latest version here>"
+            "phpcr/phpcr-shell": "<latest version here>"
```
Comment (dantleech) [inline phpcr-shell/installation.rst]: I wonder if we should say `@beta` here?

Comment (dbu) [inline phpcr-shell/installation.rst]: i did `composer require phpcr/phpcr-shell` and it automatically resolved to `~1.0@dev`. shall we just propose composer instead?

Comment (dantleech) [inline phpcr-shell/installation.rst]: Yeah, I think that would be more user friendly :)

**Comment by dbu** [comment]
changed to run composer instead of editing composer.json

File: phpcr-shell/installation.rst [hunk 2]
```diff
@@ -35,24 +35,17 @@ If you are using a Symfony2 application and a version of `DoctrinePHPCRBundle
...
 
-    { 
-        ...
-        require: {
-            ...
-            "phpcr-shell": "<latest version here>"
-        }
-        ...
-    }
+    $ composer.phar require phpcr/phpcr-shell
```
Comment (dantleech) [inline phpcr-shell/installation.rst]: Do people generally use `composer.phar`? I always simply have `composer`

File: phpcr-shell/installation.rst [hunk 3]
```diff
@@ -35,24 +35,17 @@ If you are using a Symfony2 application and a version of `DoctrinePHPCRBundle
...
 
-    { 
-        ...
-        require: {
-            ...
-            "phpcr-shell": "<latest version here>"
-        }
-        ...
-    }
+    $ composer require phpcr/phpcr-shell
```
Comment (dbu) [inline phpcr-shell/installation.rst]: you are right. http://symfony.com/doc/current/book/installation.html#installing-composer-globally recommends this way

**Pull request merged**